detail.php

```diff
@@ -21,19 +21,13 @@
         <?php echo hsc(tpl_img_getTag('IPTC.Headline',$IMG))?>
         [<?php echo strip_tags($conf['title'])?>]
     </title>
-    <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/bootstrap@5.1.3/dist/css/bootstrap.min.css" />
+    <link href='https://fonts.googleapis.com/css?family=Montserrat:400,700' rel='stylesheet' type='text/css' />
```

Comment:
Like I said, the intention is to allow users to select another font already available to them. Not to load additional fonts from ``googleapis.com``. So this is OK. It would only become DEAD Code if the user is choosing a different font.